### PR TITLE
docs: update docs to use named import and create example.js

### PR DESCRIPTION
**Pull request by straker**
Also made the docs consistent between packages (same url, same structure, etc.) and made the tests use the named import

### CODE REVIEW

File: packages/puppeteer/README.md [hunk 1]
```diff
@@ -25,12 +25,14 @@ const puppeteer = require('puppeteer');
 (async () => {
   const browser = await puppeteer.launch();
   const page = await browser.newPage();
-  await page.setBypassCSP(true);
```
Comment (straker) [inline packages/puppeteer/README.md:28]: Not sure why we'd want this as part of our docs so removed it

File: packages/webdriverio/README.md [hunk 2]
```diff
@@ -33,26 +33,29 @@ This module uses a chainable API to assist in injecting, configuring, and analyz
...
 ```js
-const AxeBuilder = require('@axe-core/webdriverio').default;
+const { AxeBuilder } = require('@axe-core/webdriverio');
 const { remote } = require('webdriverio');
 
 (async () => {
   const client = await remote({
     logLevel: 'error',
     capabilities: {
-      browserName: 'firefox'
```
Comment (straker) [inline packages/webdriverio/README.md:43]: Firefox requires a binary that isn't installed by default on macs, so switched to chrome

File: packages/webdriverjs/src/index.ts [hunk 3]
```diff
@@ -267,8 +267,4 @@ class AxeBuilder {
   }
 }
 
-if (typeof module === 'object') {
-  exports = module.exports = AxeBuilder;
-}
```
Comment (straker) [inline packages/webdriverjs/src/index.ts:272]: I don't think this is necessary anymore so removed it

File: packages/puppeteer/example.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  await page.goto('https://dequeuniversity.com/demo/mars/');
+
+  try {
+    const results = await new AxePuppeteer(page).analyze();
+    console.log(results);
+  } catch (e) {
+    console.error(e);
+  }
+
+  await page.close();
```
Comment (Zidious) [inline packages/puppeteer/example.js:16]: NDB - closing the browser should close all open pages
```suggestion

```

File: packages/playwright/example.js [hunk 5]
```diff
@@ -1,12 +1,17 @@
-const AxeBuilder = require('@axe-core/playwright').default;
+const { AxeBuilder } = require('@axe-core/playwright');
 const playwright = require('playwright');
```
Comment (Zidious) [inline packages/playwright/example.js:2]: playwright is outdated (I know this was already here) we should be using playwright-core: 

```suggestion
const playwright = require('playwright-core');
```

File: packages/playwright/example.js [hunk 6]
```diff
@@ -1,12 +1,17 @@
...
 
 (async () => {
   const browser = await playwright.chromium.launch();
```
Comment (Zidious) [inline packages/playwright/example.js:5]: As the other examples are in headless we should also add headless here too: 

```suggestion
  const browser = await playwright.chromium.launch({ headless: true });
```

**Review by Zidious — DISMISSED** [review]
LGTM - left a few comments inline nothing blocking though

File: packages/playwright/README.md [hunk 7]
```diff
@@ -6,33 +6,9 @@
 
 Install [Node.js](https://docs.npmjs.com/getting-started/installing-node) if you haven't already.
 
-Install Playwright:
+Install Playwright: `npm install playwright`
 
-NPM:
```
Comment (straker) [inline packages/playwright/README.md:11]: No need to give yarn and npm examples so made this the consistent style for all the docs.

File: packages/puppeteer/README.md [hunk 8]
```diff
@@ -8,9 +8,9 @@ Previous versions of this program were maintained at [dequelabs/axe-puppeteer](h
 
 Install [Node.js](https://docs.npmjs.com/getting-started/installing-node) if you haven't already.
 
-Install Puppeteer: `npm install puppeteer --no-save`
```
Comment (straker) [inline packages/puppeteer/README.md:11]: Not sure why we are promoting `--no-save` option here when it's a required dependency to run

**Review by Zidious — APPROVED** [review]

**Pull request merged**